osquery/logger/plugins/logger_plugin_util.h

```diff
@@ -0,0 +1,151 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
```

Comment:
I don't mind the class name being `BufferedLogForwarder`, I was more referring to the filename itself. We don't have perfect 1:1 parity in class name to file name (nor do we have much consistency around keeping one class in a file, etc).